quickstarts/microsoft.confidentialledger/confidential-ledger-create/README.md

```diff
@@ -0,0 +1,21 @@
+# Create Attestation provider
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-confidential-ledger-create/PublicLastTestDate.svg)
```

Comment:
need to update the links for the new location - see the pipeline results, you can copy/paste...